web/packages/teleterm/src/services/pty/types.ts

```diff
@@ -99,3 +99,9 @@ export type PtyCommand =
   | TshLoginCommand
   | TshKubeLoginCommand
   | GatewayCliClientCommand;
+
+export type SshOptions = {
+  // noResume disables SSH connection resumption when running `tsh ssh` (by
+  // adding the `--no-resume` option).
```

Comment:
Let's use JSDoc format:

```suggestion
  /**
   * Disables SSH connection resumption when running `tsh ssh`
   * (by adding the `--no-resume` option).
   */
```